3/T4/index.js

```diff
@@ -0,0 +1,59 @@
+const min = document.getElementById('min');
+const max = document.getElementById('max');
+const buttonFindFriendlyNumbers = document.getElementById('find');
+
+buttonFindFriendlyNumbers.addEventListener('click',() =>{
+    if(+min.value < +max.value){
+        if(+min.value >0&&+max.value >0){
+            findFriendlyNumbers(+min.value,+max.value);
+        }else{
+            alert('The numbers should be > 0');
+            min.value = '';
+            max.value = '';
+        }
+    }else{
+        alert('The beginning of the range cannot be smaller than the end');
+        min.value = '';
+        max.value = '';
+    }
+})
+
+function findFriendlyNumbers (min,max) {
+    let arrFriendlyNumbers = [];
+    for (let i = min; i <= max; i++){
+        let arrDividerFirst = getDividerNumber(i);
+        let sumArrDividerF = getSumArrDivider(arrDividerFirst);
+        if(sumArrDividerF < max){
+            let arrDividerSecond = getDividerNumber(sumArrDividerF);
+            let sumArrDividerS = getSumArrDivider(arrDividerSecond);
+            if(i === sumArrDividerS&&sumArrDividerF > sumArrDividerS?true:false){
```

Comment:
`if (i === sumArrDividerS && sumArrDividerF > sumArrDividerS) {`

this is absolutely identical. + prettify code

Reply:
done